### ISSUE TITLE
Find panel font size option does not work as intended?

### ISSUE THREAD

**Issue by michaelblyons** [oivva/boxy#48]
I am trying to increase the font size of the text I type into the "Find" panel. I was using the ["Find Panel Font" options][ops] from the Settings wiki:

```js
// Increase font size of the find panel to 12 px
"theme_find_panel_font_md": true,
```

### Expected behavior:

- Find panel input text size increases
- Find panel button text size increases (although I would avoid this if possible)

### Actual behavior:

- Status bar text size increases
- Find panel button text size increases
- Find panel input text size stays the same

It would be nice if I could change the input boxes' font size independent of the buttons. The buttons look good already, but the input box text is very small on OSX<sup>1</sup>. There's plenty of room in the box for an increase in text size.

-----

<sup>1</sup>Windows looks fine.

[ops]: https://github.com/oivva/boxy/wiki/Settings#find-panel-font

**Comment by oivva** [comment]
Thanks for reporting the issue. 

> Status bar text size increases
Agree, I'll fix this behavior in 2.0.1.

> It would be nice if I could change the input boxes' font size independent of the buttons. The buttons look good already, but the input box text is very small on OSX1. There's plenty of room in the box for an increase in text size.

Due to Sublime Text limitations:
1. Input labels has the same font size as button labels. 
2. I can't change font size of the input text via options. But you can do this manually: 
    * Extract theme via [Package Resource Viewer](https://github.com/skuroda/PackageResourceViewer)
    * Open the file: **`Packages/Boxy Theme/widgets/Widget - <Theme>.sublime-settings`**
    * Change **`"font_size": 11`** 

**Note**: 
* You should repeat this after each theme update
* This font size also affects Quick Panel and Console

P.S. I have plans to investigate if I can create addons for such cases.

**Comment by oivva** [comment]
@michaelblyons Starting from now you can use this [addons](https://github.com/search?q=user%3Aoivva+boxy-addon-widget) to change the font size of the text inputs.
